lang/LangPrimSource/PyrArrayPrimitives.cpp

```diff
@@ -2369,12 +2383,142 @@ int prArrayUnlace(struct VMGlobals *g, int numArgsPushed)
 	return errNone;
 }
 
+extern bool compiledOK;
+PyrSymbol* symb_unixCmdAction;
+
+static void string_popen_thread_func(struct sc_process *process)
```

Comment:
From a quick read on static (c) functions, what I got is that they are local to the file. So my reasoning was that it wouldn't make sense to export it from a .hpp file, therefore I copied it here. Does this make sense ?